query-connector/src/app/queryBuilding/utils.ts

```diff
@@ -94,3 +116,63 @@ export function filterSearchByCategoryAndCondition(
 export function formatDiseaseDisplay(diseaseName: string) {
```

Comment:
we have a utility function for this: https://github.com/CDCgov/dibbs-query-connector/blob/main/query-connector/src/app/queryBuilding/utils.ts#L94.

I kinda wonder if we should do this cleanup at the DB creation level, at least for APHL code, since we otherwise need to include this in other places.

Reply:
per convo today, this will be covered in a future ticket/discussion!